### PR TITLE
table size value validation

### PR DESCRIPTION
**Pull request by jingweicb**
Fixes # .

### CODE REVIEW

File: configuration/configuration.go [hunk 1]
```diff
@@ -353,6 +353,14 @@ func assertConfiguration(ctx context.Context, config *Configuration) error {
 		return errors.New("serial_block_workers must be > 0")
 	}
 
+	if config.TableSize != nil && (*config.TableSize < 2 || *config.TableSize > 100) {
```
Comment (jingweicb) [inline configuration/configuration.go:356]: @GeekArthur point enable us to check nil, as these fields are omitempty

File: configuration/configuration.go [hunk 2]
```diff
@@ -353,6 +353,14 @@ func assertConfiguration(ctx context.Context, config *Configuration) error {
...
+		return fmt.Errorf("table_size %d range is [2, 100], please check your input", *config.TableSize)
```
Comment (GeekArthur) [inline configuration/configuration.go:357]: Probably change `table_size %d range is [2, 100]` to `table_size %d is not in the range [2, 100]`

File: configuration/configuration.go [hunk 3]
```diff
@@ -353,6 +353,14 @@ func assertConfiguration(ctx context.Context, config *Configuration) error {
...
+	}
+
+	if config.ValueLogFileSize != nil && (*config.ValueLogFileSize < 128 || *config.ValueLogFileSize > 2048) {
+		return fmt.Errorf("value_log_file_size %d range is [128, 2048], please check your input", *config.ValueLogFileSize)
```
Comment (GeekArthur) [inline configuration/configuration.go:361]: Probably change `value_log_file_size %d range is [128, 2048]` to `value_log_file_size %d is not in the range [128, 2048]`

**Review by GeekArthur — CHANGES_REQUESTED** [review]

**Review by GeekArthur — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for GeekArthur @ 2022-08-16 22:51:31 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

File: cmd/root.go [hunk 4]
```diff
@@ -269,15 +269,15 @@ default values.`,
 	checkDataCmd.Flags().Int64Var(
 		&tableSize,
 		"table-size",
-		0,
+		-1,
 		"Table-size configures the TableSize for badger DB. If table-size != 0, this will override the table_size from configuration file",
```
Comment (GeekArthur) [inline cmd/root.go:273]: `table-size != -1` instead of `table-size != 0`?

**Review by GeekArthur — APPROVED** [review]

**Pull request merged**